deploy/docker-swarm/README.md

```diff
@@ -0,0 +1,97 @@
+# Docker Swarm Hetzner CSI plugin
+
+Currently in Beta. Please consult the Docker Swarm documentation
```

Comment:
What exactly is in `beta`? The _Docker Swarm CSI_ support or the _hcloud-csi-driver Docker Swarm_ support?

Reply:
Good that you point this out.

According to https://docs.docker.com/engine/release-notes/23.0/ the CSI support is "experimental" but I don't  think this is to be read as "may be removed again".

I added this line to this file to mention that there is no guarantee that this plugin + Docker Swarm CSI will work 100% for everyone. This is a new feature after all.